core/domain/cve.go

```diff
@@ -0,0 +1,9 @@
+package domain
+
+type CVE struct {
+	ImageID            string
+	SBOMCreatorVersion string
+	CVEScannerVersion  string
+	CVEDBVersion       string
+	Content            []byte
+}
```

Comment:
question: what is CVE?

Is it a list of vulnerabilities? If so, then, as Ben said, CVE is a pretty specific name. Moreover it usually refers to one vulnerability, not a collection of them, so if it’s a list of vulnerabilities, we should name it appropriately.